pkg/local_workflows/code_workflow.go

```diff
@@ -50,12 +82,25 @@ func codeWorkflowEntryPoint(invocationCtx workflow.InvocationContext, _ []workfl
 	// get necessary objects from invocation context
 	config := invocationCtx.GetConfiguration()
 	logger := invocationCtx.GetEnhancedLogger()
+	sastEnabled := config.GetBool(ConfigurationSastEnabled)
 	ignoresFeatureFlag := config.GetBool(configuration.FF_CODE_CONSISTENT_IGNORES)
 	reportEnabled := config.GetBool("report")
 
 	logger.Debug().Msg("code workflow start")
+	logger.Debug().Msgf("SAST Enabled:       %v", sastEnabled)
 	logger.Debug().Msgf("Consistent Ignores: %v", ignoresFeatureFlag)
-	logger.Debug().Msgf("Report enabled: %v", reportEnabled)
+	logger.Debug().Msgf("Report enabled:     %v", reportEnabled)
+
+	if !sastEnabled {
+		err = snyk_errors.Error{
+			ID:             "Snyk-CLI-0001",
+			Title:          "Snyk Code is not supported",
+			Classification: "ACTIONABLE",
+			Level:          "warning",
+			Detail:         "Snyk Code is not supported for org " + config.GetString(configuration.ORGANIZATION) + ": enable in Settings > Snyk Code",
+		}
```

Comment:
**suggestion:** We could include a Link to the public facing documentation here:
https://docs.snyk.io/scan-using-snyk/snyk-code/configure-snyk-code#enable-snyk-code-in-snyk-web-ui

Reply:
Done!